bin.js

```diff
@@ -61,7 +61,7 @@ var argv = minimist(process.argv.slice(2), {
   if (argv.help) {
     console.log(USAGE)
   } else if (argv.version) {
-    console.log(require('../package.json').version)
+    console.log(require('./package.json').version)
```

Comment:
oh, yeah so it's a dynamic require of a JSON blob - which is then transformed to an object and we just read the `version` property off it. Best way to get the current version :sparkles: - way easier than `fs.readFile()` + `JSON.parse()` and stuff C: